include/gauge_backup.h

```diff
@@ -0,0 +1,157 @@
+#pragma once
+
+#include <algorithm>
+#include <gauge_field.h>
+
+namespace quda
+{
+
+  struct GaugeBundleBackup {
+    GaugeField *precise;
+    GaugeField *sloppy;
+    GaugeField *precondition;
+    GaugeField *refinement;
+    GaugeField *eigensolver;
+    GaugeField *extended;
+
+    GaugeBundleBackup() :
+      precise(nullptr), sloppy(nullptr), precondition(nullptr), refinement(nullptr), eigensolver(nullptr), extended(nullptr)
```

Comment:
As noted for above, set these values inline above and use the compiler-generated default constructor

Reply:
Done in latest commit